packages/web/src/pages/pay-and-earn-page/mobile/PayAndEarnPage.tsx

```diff
@@ -38,19 +40,21 @@ type TableMetadata = {
 
 export const PayAndEarnPage = ({ tableView }: PayAndEarnPageProps) => {
   const dispatch = useDispatch()
-  const accountHasTracks = useSelector(getAccountHasTracks)
+  const { data: hasTracks } = useCurrentAccountUser({
+    select: (user) => selectAccountHasTracks(user)
```

Comment:
this approach is actually kinda cool, you can probably just do `select: selectAccountHasTracks` here any lots of other places?